examples/arithmetics/README.md

```diff
@@ -4,8 +4,11 @@
 
 The Arithmetics Example features an interpreter that you can run via cli.
 
-Ensure the complete project was properly built, otherwise run `npm install` from the root of the Langium project.
-Use `node ./bin/cli` to run the cli. Follow the instructions or use `node ./bin/cli eval <full-path-to-calc-file>`.
+* Ensure the complete project was properly built, otherwise run `npm install` from the root of the Langium project.
+* Run `npm install -g ./` from the arithmetics directory.
+* Use `arithmetics-cli` to run the cli. Follow the instructions or use `arithmetics-cli eval <full-path-to-calc-file>`.
```

Comment:
This is one way to run it, but for quick testing I find it easier to just call  `./bin/cli`. Can you add this as an alternative to the global install (affects the other examples, too)?

Reply:
I've updated the arithmetics README -- could you check? If it's okay, I update all other READMEs.